src/modules/reporting/components/reporting-dispute-form/reporting-dispute-form.jsx

```diff
@@ -26,9 +26,15 @@ export default class ReportingDisputeForm extends Component {
     accountDisputeData: PropTypes.object,
   }
 
-  static checkStake(stake, updatedValidations) {
-    if (stake === '' || stake == null || stake <= 0) {
+  static checkStake(wholeREPstake, updatedValidations, maxRep) {
```

Comment:
Given this is static could we move is to another file and add tests?

Reply:
could, but it's not used by anything else. It has errors that are specifically for this form. I want to keep it with the form.